_data/snippets.yml

```diff
@@ -51,3 +51,27 @@ environment_variables: |
 
   - differ per repository.
   - contain sensitive data, such as third-party credentials.
+contact_enterprise_support: |
+  To get in touch with us, please write a message to 
+  [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com). If possible, 
+  please include as much of the following info as you:
+   
+  - What is the problem?
```

Comment:
We maybe need to rewrite these? Because "what is the problem" is not a piece of info. So it would become

- a description of the problem
...

Alternatively, reword the preceeding sentence to something like, "answer as many o the questions as you can"